exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -40,6 +40,29 @@
    */
   HashCode submitTransaction(TransactionMessage tx);
 
+  /**
+   * Returns a number of unconfirmed transactions those are currently located in
+   * the memory pool and are waiting for acceptance to a block.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  int getUnconfirmedTransactions();
+
+  /**
+   * Returns <b>true</b> if the node is connected to the other peers.
+   * And <b>false</b> otherwise.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  boolean healthCheck();
+
+  /**
+   * Returns string containing information about Exonum, Rust and OS version.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  String getUserAgentInfo();
```

Comment:
A `String`? Isn't it a JSON we can easily convert into something better, e.g.,
```
NodeUserAgent
- exonumVersion: String
- rustVersion: String
- osVersion: String
```

Reply:
As it turned out not be in JSON but a `/`-separated String, do you think we shall go with `NodeUserAgent` and parsing or a plain String? Will clients ever need to access individual components or just log the whole thing? If the latter, which format is better — the original or the one from `#toString` implementation ("exonumVersion=x, rustVersion=y, osVersion=z")?

Reply:
I believe we shouldn't care about the string parsing in this case. And we can return it as is.

Reply:
Ok let's go for it and get rid of some extra code.